### PR TITLE
updated vs for mac tab

### LINKED ISSUE #13816: VS for mac outdated
Planning to update VS for mac instructions from Terminal to actually using VS for mac to create a [Razor Pages Web App](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/start-mvc?view=aspnetcore-2.2&tabs=visual-studio-mac#create-a-web-app-1).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a25bf940-8977-fec4-461b-542a9f550fc8
* Version Independent ID: 5492c641-af15-64b8-beaa-ddf511be73e6
* Content: [Tutorial: Get started with Razor Pages in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/razor-pages-start?view=aspnetcore-2.2&tabs=visual-studio-mac#create-a-razor-pages-web-app-1)
* Content Source: [aspnetcore/tutorials/razor-pages/razor-pages-start.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/razor-pages/razor-pages-start.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by LouisT123**
fixes #13816

### CODE REVIEW

**Comment by LouisT123** [comment]
@Rick-Anderson 
cc/ @anangaur

File: aspnetcore/tutorials/razor-pages/razor-pages-start.md [hunk 1]
```diff
@@ -87,15 +87,22 @@ At the end of this tutorial, you'll have a working Razor Pages web app that you'
...
+![macOS New solution](../first-mvc-app/start-mvc/_static/new_project_vsmac.png)
 
-```console
-dotnet new webapp -o RazorPagesMovie
-```
+* Select **.NET Core** > **App** > **Web Application** > **Next**.
+
+  ![macOS New project dialog](razor-pages-start/_static/webapp.png)
+
+* In the **Configure your new ASP.NET Core Web API** dialog, set the  **Target Framework** of **.NET Core 2.0**.
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/razor-pages/razor-pages-start.md]: In the **Configure your new ASP.NET Core Web API** dialog, set the  **Target Framework** to **.NET Core 2.2**.

Comment (Rick-Anderson) [reply aspnetcore/tutorials/razor-pages/razor-pages-start.md]: You want 2.2, not 2.0.

Comment (LouisT123) [reply aspnetcore/tutorials/razor-pages/razor-pages-start.md]: My bad there, thanks @Rick-Anderson

**Comment by Rick-Anderson** [comment]
BTW, you should be able to accept my suggestion from the browser so you don't have to make a commit.

**Comment by Rick-Anderson** [comment]
Look at the [preview](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/razor-pages-start?view=aspnetcore-3.0&branch=pr-en-us-13829&tabs=visual-studio-mac)

You made the update in the 3.0 portion of the tutorial, so I updated it to 3.0 (see my last commit). Did you want to make the same change to the 2.2 version of the tutorial? That's the default version and what people see by default until 3.0 GA's  next month.

**Comment by Rick-Anderson** [comment]
BTW, you can get the preview URL from the details link
image

It's always good to review the internal build

**Comment by LouisT123** [comment]
@Rick-Anderson I want to update the 2.2 version for now,since I don't have a screenshot showing version 3.0 being selected but this could also work.

**Comment by LouisT123** [comment]
> @Rick-Anderson I want to update the 2.2 version for now,since I don't have a screenshot showing version 3.0 being selected but this could also work.

actually just updated my internal preview i got 3.0 now so i will try to get a new screenshot

**Pull request merged**